src/components/WebglBox.tsx

```diff
@@ -44,7 +44,7 @@ function WebglBox({ userId, experienceData }: WebglProps): JSX.Element {
           zIndex: 0,
         }}
       />
-      <Overlay userId={userId} experienceData={experienceData} />
+      <Overlay userId={userId} experienceData={experienceData} tabIndex={1} />
```

Comment:
Azure Static Web Apps CI/CD detected an error when trying to compile this branch.

**Type error: Type '{ userId: string; experienceData: ExperienceData; tabIndex: number; }' is not assignable to type 'IntrinsicAttributes & OverlayProps'.
  Property 'tabIndex' does not exist on type 'IntrinsicAttributes & OverlayProps'.**